README.md

```diff
@@ -44,14 +44,20 @@ Explore our [demo app](https://demo.airbyte.io/).
 
 ### Run Airbyte locally
 
-You can run Airbyte locally with Docker.
+You can run Airbyte locally with Docker or Podman.
 
 ```bash
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
 docker compose up
 ```
 
+```bash
+git clone https://github.com/airbytehq/airbyte.git
+cd airbyte
+env $(cat .env.podman) podman-compose -f docker-compose.podman.yaml up
```

Comment:
What happen if you use .env and docker-compose.yaml file instead?

Reply:
Another issue that arises is the manner in which references to variables are defined.